core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.css

```diff
@@ -4,194 +4,332 @@
 
-.oppia-lesson-info-card .oppia-exploration-header {
-  font-size: 17px;
-  font-weight: normal;
-  margin: 0 0 10px 0;
-  padding: 0 20px 0;
+.oppia-info-card-tooltip-more {
+  color: blue;
 }
 
-.oppia-lesson-info-card .oppia-non-story-mode .oppia-exploration-header {
-  margin: unset;
-  padding: 20px 20px 0;
+.oppia-info-card-bottom-row .sharing-links-container {
+  display: flex;
+  justify-content: center;
+  margin-top: 20px;
 }
 
-.oppia-lesson-info-card {
-  border-bottom-left-radius: 4px;
-  border-bottom-right-radius: 4px;
-  padding: 0 0 6px 0;
-  text-align: left;
+/* This is necessary so that long usernames don't overflow the tooltip. */
+.oppia-lesson-exploration-progress-info .tooltip-inner {
+  max-width: none;
 }
 
-/* Hide scrollbar for IE, Edge and Firefox */
-.oppia-lesson-info-content-inner-container {
-  max-height: 260px;
-  -ms-overflow-style: none;  /* IE and Edge */
-  overflow-y: auto;
-  scrollbar-width: none;  /* Firefox */
+.oppia-info-card-tag-icon {
+  padding-top: 6px;
+  width: 28px;
 }
 
-.oppia-non-story-mode .oppia-lesson-info-content-inner-container {
-  max-height: 220px;
-  overflow-y: auto;
+.oppia-info-card-exploration-contributors-profile {
+  justify-content: flex-end;
+  padding: 0;
 }
 
-/* Hide scrollbar for Chrome, Safari and Opera */
-.oppia-lesson-info-content-inner-container::-webkit-scrollbar {
-  display: none;
+.oppia-info-card-community-editable-icon {
+  color: #009688;
 }
 
-.oppia-lesson-info-card .oppia-lesson-exploration-progress-info {
-  padding: 0 20px 20px 20px;
+.learn-info-card-star {
+  margin: 5px
 }
 
-.oppia-lesson-info-card .oppia-story-title {
-  color: #3b3b3b;
-  font-family: "Capriola", "Roboto", Arial, sans-serif;
-  font-size: 13px;
-  margin-bottom: -10px;
-  padding: 20px 20px 0;
-  text-align: left;
+.oppia-lesson-exploration-progress-info .card-metrics li .far {
+  font-size: 24px;
+  margin: 5px;
+  vertical-align: middle;
 }
 
-.oppia-lesson-info-card .oppia-lesson-info-content {
-  padding-bottom: 0;
+.oppia-info-card-bottom-row ul li a img {
+  height: 20px;
+  width: 20px;
 }
 
-.oppia-lesson-info-card hr {
-  margin-left: 20px;
-  margin-right: 20px;
+.named-contributors-container .contributor-0 {
+  list-style: none;
+  position: absolute;
+  z-index: 2;
 }
 
-.oppia-lesson-info-card .oppia-non-story-mode hr {
-  margin-top: 8px;
+.named-contributors-container .contributor-1 {
+  list-style: none;
+  margin-left: 14px;
+  position: absolute;
 }
 
-.oppia-lesson-info-card .oppia-exploration-description {
-  color: #333;
-  font-size: 15px;
-  font-weight: normal;
-  padding-bottom: 10px;
+.oppia-lesson-exploration-progress-info .card-metrics {
+  box-sizing: border-box;
+  font-size: 14px;
+  list-style-type: none;
+  margin-top: 15px;
+  max-width: 100%;
+  padding: 0;
+  text-align: center;
 }
 
-.oppia-lesson-info-card .progress {
-  background: #f0ffff;
-  border: 1px solid #707070;
-  border-radius: 10px;
+.oppia-info-card-bottom-row md-chips md-chip {
```

Comment:
I removed this CSS rule because, upon inspecting the bottom-row div and the elements enclosed within it (e.g. the ngbTooltip, etc.), I couldn't find any md-chips elements. After removing the rule, I couldn't find a difference either.